spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -4067,8 +4074,10 @@ def _handle_completions(self):
 
         self.document_did_change(text)
 
+        is_backspace = (
+            self._count_backspace >= self.automatic_completions_after_chars)

```

Comment:
Does this mean that you have to press Backspace at least `automatic_completions_after_chars` to get completions for it?

What happens if you press Backspace only one time?

Reply:
This is true! It should be fixed by the next commit